Bixal/move.mil
# Feature 156620768 update themeing txt w image align if else title

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- updates our alignments to reflect left and right...correctly 👍 
- updates our field assignments to pass through alignment values from selected inputs in content admin view
- if there is an image on and a title, display our title on the media-content
- if there is no image on and a title, display our title on the media-content
- if there is a title with a link, wrap our title and our image

## Testing

To verify the changes proposed in this pull request…

1. pull changes to your local
2. cd /move-mil/web/themes/custom/move_mil
3. npm run build
4. create a basic page with alternating left or right image aligned text w/image paragraphs to match the following view

![screencapture-move-mil-localhost-8000-node-1-2018-04-10-17_18_45](https://user-images.githubusercontent.com/37077057/38584810-83118dcc-3ce5-11e8-8149-a15ac163fff2.png)
